package/yast2-bootloader.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Apr 12 13:29:54 UTC 2016 - jreidinger@suse.com
+
+- do not propose nor running grub2-install on bare metal ppc
```

Comment:
NP: Inconsistencies found:

- capitalization, the item below starts with `Do not ...` and it should be `nor run`
- the changes below use `POWER`, not `ppc` (I'm not sure which one should be preferred)